libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -611,7 +611,8 @@
       "calculationType": "Beregningstype",
       "messageType": "Meddelelsestype",
       "gridArea": "Netområde",
-      "status": "Status"
+      "status": "Status",
+      "energySupplier": "Elleverandør"
```

Comment:
Question: not possible to use shared here as well?

Reply:
This is the topmost `shared` in `eSett`. Couldn't find other `shared` higher up that contain `Elleverandør`. Did you mean to add it?